pkg/assets/bindata.go

```diff
@@ -0,0 +1,6928 @@
+// Code generated for package assets by go-bindata DO NOT EDIT. (@generated)
```

Comment:
Please remove this file

Reply:
Done. I did the rebase and I'm not sure why this remained as part of the PR. Sorry!